Source Code.txt

```diff
@@ -1,87 +0,0 @@
----AndroidManifest.xml---
```

Comment:
Moved txt file to a markdown also as README.md such that GitHub can show previews